zosia_site/terraform.tfvars

```diff
@@ -0,0 +1,11 @@
+project_id = ""
+region     = "europe-central2"
+
+db_settings = {
+  username         = "zosia-admin"
+  instance_name    = "zosia-db"
+  db_name          = "zosia"
+  database_tier    = "db-f1-micro"
+  database_edition = "ENTERPRISE"
+  database_version = "POSTGRES_15"
+}
```

Comment:
All those values could have been defined as [local variables](https://developer.hashicorp.com/terraform/language/values/locals). All those values could have been defined as local variables. Then you can get rid of the `.tfvars` file and `variables.tf`.